tests/integration/test_perl_samples.py

```diff
@@ -24,9 +24,9 @@
     'array-content-test.conf': {'domain':
                                     ['b0fh.org', 'l0pht.com', 'infonexus.com']},
     'unquoted-values-with-whitespaces.conf': {'option': 'value with whitespaces'},
-    'multiline-option-test.conf': {'command': "ssh -f -g orpheus.0x49.org           "
```

Comment:
Is it useful or important to strip trailing whitespaces?

Reply:
This is to make the test behave as expected as reported in #39, where for multi-line values, the spaces between words is normalized to a single space

Reply:
(Behavior w.r.t. trailing whitespaces should be unchanged!)

Reply:
Ah, right! So the trailing whitespaces are trimmed only in the multi-line situation?

Considering this is a fix rather than a feature, we do not need to conditionally preserve old behaviour, right?

Reply:
We just want to preserve behavior single-line values so it matches the behavior of the perl parser-- so `key multi\tword \t value` would become `{"key": "multi\tword \t value"}` but `key multi\\\nline \t value` becomes `{"key": "multi line value"}`